pkg/bpmn_engine/engine_jobs_activated.go

```diff
@@ -18,16 +19,32 @@ type activatedJob struct {
 	processDefinitionKey     int64
 	elementId                string
 	createdAt                time.Time
+	scope                    variable_scope.VarScope
+	localScope               variable_scope.VarScope
 }
 
 // ActivatedJob represents an abstraction for the activated job
 // don't forget to call Fail or Complete when your task worker job is complete or not.
 type ActivatedJob interface {
-	ProcessInstance
+	// GetInstanceKey get instance key from processInfo
+	GetInstanceKey() int64
+	// GetCreatedAt  get job create time
+	GetCreatedAt() time.Time
+	// GetState get instance state
+	GetState() process_instance.State
 
 	// GetKey the key, a unique identifier for the job
 	GetKey() int64
 
+	// GetVariable the varaible from variable scope  include local scope
+	GetVariable(key string) interface{}
+
+	// SetVariable set variable into variable scope
+	SetVariable(key string, value interface{})
+
+	// SetVariableLocal set variable into local variable scope
+	SetVariableLocal(key string, value interface{})
```

Comment:
Reading the BPMN documentation from Camunda, I don't see a need, for introducing a local scope to lib-bpmn-engine users. This just gives more confusion. Please remove such.

Reply:
Good point, it's not a good way to introduce SetVariableLocal. I have removed this function.